PureBasicIDE/ProcedureBrowser.pb

```diff
@@ -186,7 +191,11 @@ Procedure UpdateProcedureList()
       EndIf
     Next ProcedureList()
     
-    CompilerIf #CompileLinux
+    ;- Author : mk-soft
+    ;  Date   : 22.05.2020
+    ;  Bugfix : Linux Crash with g_object. Disabale optimize
+    
+    CompilerIf 0; #CompileLinux
```

Comment:
If the code within the disabled `CompilerIf` block no longer makes sense, the complete block should be removed.